WatchApp Extension/Startup/HostingController.swift

```diff
@@ -8,10 +8,10 @@
 import Combine
 import CoreData
 import Foundation
+import SimplistsKit
 import StoreKit
 import SwiftUI
 import WatchKit
-import SimplistsWatchKit
```

Comment:
Changed `SimplistsKit` framework to be multi-platform. Deleted `SimplistsWatchKit` target.